src/main/scala/gitbucket/core/controller/api/ApiIssueCommentControllerBase.scala

```diff
@@ -71,8 +72,18 @@ trait ApiIssueCommentControllerBase extends ControllerBase {
 
   /*
    * vi. Delete a comment
-   * https://developer.github.com/v3/issues/comments/#delete-a-comment
+   * https://docs.github.com/en/rest/reference/issues#delete-an-issue-comment
+   *
    */
+  delete("/api/v3/repos/{owner}/{repo}/issues/comments/:id")(readableUsersOnly { repository =>
+    (for {
+      commentId <- params("id").toIntOpt
+      comment <- getComment(repository.owner, repository.name, commentId.toString)
+      issue <- getIssue(repository.owner, repository.name, comment.issueId.toString)
```

Comment:
I'm thinking of doing a permissions check here using `isEditable`, is that a valid approach?

Reply:
Yeah, checking whether the user is allowed to delete the comment is necessary. We need to do the same check what we are doing here:
https://github.com/gitbucket/gitbucket/blob/73d05aefadb3e9633818c6dfe40c2a4923f93c6e/src/main/scala/gitbucket/core/controller/IssuesController.scala#L237-L246